src/Descriptor.hpp

```diff
@@ -108,4 +108,15 @@ class Descriptor {
 
     void processing_command(bool is_processing) noexcept { processing_command_ = is_processing; }
     [[nodiscard]] bool processing_command() const noexcept { return processing_command_; }
+
+    [[nodiscard]] CHAR_DATA *character() const noexcept { return character_; }
+    void character(CHAR_DATA *character) noexcept { character_ = character; }
+    [[nodiscard]] CHAR_DATA *original() const noexcept { return original_; }
+    // do_ prefix because switch is a C keyword
+    void do_switch(CHAR_DATA *victim);
+    // do_ prefix because return is a C keyword
+    void do_return();
+    // Return the real-life "person" player character behind this descriptor.
```

Comment:
yeah makes sense. And in this case, it's returning full CHAR_DATA, not just the PC_DATA, which it contains if the CD is a person :)

Reply:
I nicked the name from a commented line in a few places. I _think_ it's safe to assume `person()->pc_data` is non-null...I dunno :)